website/frontend/src/api/initApi.ts

```diff
@@ -20,15 +32,15 @@ class NodeApi {
   readonly subscriptions: Record<string, UnsubscribePromise> = {};
 
   constructor(address = 'ws://localhost:9944') {
```

Comment:
How about to move getting this one from consts?

Reply:
It's a standalone task.